frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.ts

```diff
@@ -42,6 +49,11 @@ export class ModelDiagramDialogComponent {
 
   search = '';
 
+  @ViewChild('codeBlock', { static: false }) codeBlock!: ElementRef;
```

Comment:
```suggestion
  @ViewChild('codeBlock') codeBlock!: ElementRef;
```
`false` should be the default value here, so we can omit it